ui/cryptomaterial/wallet_clickable_list.go

```diff
@@ -0,0 +1,116 @@
+package cryptomaterial
+
+import (
+	"gioui.org/layout"
+	"gioui.org/unit"
+	"github.com/crypto-power/cryptopower/libwallet/utils"
+)
+
+type ClickedItem struct {
+	AssetType utils.AssetType
+	Index     int
+}
+
+type WalletClickableList struct {
+	layout.List
+	theme           *Theme
+	clickables      []*Clickable
+	Radius          CornerRadius // this radius is used by the clickable
+	selectedItem    int
+	DividerHeight   unit.Dp
+	IsShadowEnabled bool
+	IsHoverable     bool
+	ItemIDs         map[*Clickable]ClickedItem
+}
```

Comment:
Whats the difference between this and `ClickableList`? they look similar but for the `ItemIDs`. it looks like a repeat.

Reply:
This is specific to the wallet list only, since it has the complexity of tracking the asset type and wallet ID/index